.github/workflows/uploadFirrtlReleaseArtifacts.yml

```diff
@@ -27,10 +27,13 @@ jobs:
             --exclude=circt-full-sources.tar.gz \
             -czf \
             circt-full-sources.tar.gz .
+          shasum -a 256 circt-full-sources.tar.gz | cut -d ' ' -f1 > circt-full-sources.tar.gz.sha256
```

Comment:
Curious why using perl's `shasum` instead of coreutils' `sha56sum`? But whatever works :+1: .

Reply:
`sha256sum` wasn't available by default on the runners, maybe just the Mac runner actually? No coreutils by default on Mac presumably? I'm not sure but I picked it because (at some point) it worked on both Windows and Linux for the sha256sum of the binary artifacts. I'm happy to switch this--no real opinion.

Reply:
Nah, that's great and thanks for the context I was curious especially if doing this sort of thing elsewhere.

Yes I'd expect mac runner to maybe not have gnu coreutils, totally.  Fine as-is, especially if using coreutils requires special-casing other platforms.  I'm sure `shasum` is fine :+1:.